pkg/pool-weighted/test/LiquidityBootstrappingPool.test.ts

```diff
@@ -1,2 +1,161 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { BigNumber } from 'ethers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+import { actionId } from '@balancer-labs/v2-helpers/src/models/misc/actions';
+import { fp } from '@balancer-labs/v2-helpers/src/numbers';
+import { MINUTE, advanceTime, currentTimestamp } from '@balancer-labs/v2-helpers/src/time';
+
+import TokenList from '@balancer-labs/v2-helpers/src/models/tokens/TokenList';
+import WeightedPool from '@balancer-labs/v2-helpers/src/models/pools/weighted/WeightedPool';
+
+import { itBehavesAsWeightedPool } from './BaseWeightedPool.behavior';
+
 // eslint-disable-next-line @typescript-eslint/no-empty-function
-describe('LiquidityBootstrappingPool', function () {});
+describe('LiquidityBootstrappingPool', function () {
+  describe('as a 2-token weighted pool', () => {
+    itBehavesAsWeightedPool(2);
+  });
+
+  describe('as a 3-token weighted pool', () => {
+    itBehavesAsWeightedPool(3);
+  });
```

Comment:
This is just testing regular weighted pools. Did you mean to add an extra argument?

Reply:
No, I meant to make sure nothing in the derived class broke any standard behavior (since it does override things). Unnecessary?